pyweather.py

```diff
@@ -1909,7 +1956,156 @@ def radar_clearImages():
     summary_forecastIterations = summary_forecastIterations + 1
     if summary_forecastIterations == 5:
         break
+
 print("")
+
+logger.debug("yesterdayJSON loaded with: %s" % yesterdayJSON)
+yesterday_json = json.loads(yesterdayJSON.text)
+if jsonVerbosity == True:
+    logger.debug("yesterday_json: %s" % yesterday_json)
+else:
+    logger.debug("Loaded 1 JSON.")
+yesterday_date = yesterday_json['history']['date']['pretty']
+print(Fore.YELLOW + "Here's yesterday's weather for " + Fore.CYAN +

```

Comment:
Make sure you add a conditional that'll only show yesterday's weather on summary if the option to do so is enabled.

Reply:
Done